shell-completion/bash/systemd-analyze

```diff
@@ -30,6 +30,15 @@ __get_machines() {
         machinectl list --no-legend --no-pager | { while read a b; do echo " $a"; done; };
 }
 
+__get_syscall_sets() {
+        local line
+        systemd-analyze syscall-filter --no-pager | while IFS= read -r line; do
```

Comment:
actually, I am pretty sure the `--no-pager` thing is just cargo-culting here... it's the implied default if our tools detect they are run as part of a shell pipeline (i.e. stdout is not a tty)... hence it should be safe to remove it here and everywhere else where we make it part of a shell pipeline..

Reply:
Yeah. We can probably do that in one fell swoop though. Doesn't matter to much here.